front/app/components/UI/QuillEditedContent/QuillEditedContent.test.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { render, screen, waitFor } from 'utils/testUtils/rtl';
+import QuillEditedContent from '.';
+import 'jest-styled-components';
```

Comment:
Why do you need this?

Reply:
Copy/paste error from another test. Removing :)